packages/webapp/src/utils/credits-application.js

```diff
@@ -0,0 +1,19 @@
+import constants from './constants.js'
+
+const getCreditsAppReference = session => session.get(constants.redisKeys.CREDITS_PURCHASE_APPLICATION_REFERENCE) || ''
+
+// Credits Estimator's object schema must match the expected payload format for the Operator application
+export default (session, account) => {
+  return {
+    creditsEstimation: {
+      applicant: {
+        firstName: account.idTokenClaims.firstName,
```

Comment:
Won't need firstName, lastName and emailAddress in the payload so can remove them. We only need the contactId